x-pack/solutions/observability/plugins/synthetics/server/tasks/sync_private_locations_monitors_task.ts

```diff
@@ -108,6 +116,7 @@ export class SyncPrivateLocationMonitorsTask {
             encryptedSavedObjects,
           });
         }
+        packagePoliciesRequireReformatting = false;
```

Comment:
I only set this variable to false after the syncGlobalParams function completes successfully. However, I'm concerned about this. I'd like a second opinion @shahzad31. On one hand, I'd love to be able to retry after an errors. On the other, I'm not sure we know which errors are retry-able versus what could cause the task to always fail consistently, resulting in a scenario in which the variable is always `packagePoliciesRequireReformatting: true` and try to reformat it endlessly. Maybe I should add a retry count?

Reply:
if you want it to be not infinite process with errors, i will suggest implement retries 

`packagePoliciesRequireReformatting : {done: true, maxRetries: 3}`

that way you can exhaust retries one by one